MechJeb2/MechJebModuleAscentGuidance.cs

```diff
@@ -447,9 +447,12 @@ private void ShowAutoWarpGUIElements()
                                 vesselState.latitude,
                                 vesselState.celestialLongitude,
                                 core.target.TargetOrbit.LAN - autopilot.launchLANDifference,
-                                core.target.TargetOrbit.inclination
+                                core.target.TargetOrbit.inclination,
+                                out bool launchingNorth
                                 )
                             );
+                    // Fix up the sign of the inclination to match the window MinimumTimeToPlane selected
+                    desiredInclination = launchingNorth ? Math.Abs(desiredInclination) : -Math.Abs(desiredInclination);
```

Comment:
there's also fixup code down here:

https://github.com/MuMech/MechJeb2/blob/41272b4a3a328013127b7df7e13fdefd31cca9ef/MechJeb2/MechJebModuleAscentGuidance.cs#L497-L503=

that now seems redundant and could be deleted after this change?

Reply:
Done.